modules/scripts/startup-script/templates/startup-script-custom.tpl

```diff
@@ -1,12 +1,18 @@
 
 
 stdlib::run_playbook() {
+  if [ -d /usr/local/ghpc-venv ]; then
+    . /usr/local/ghpc-venv/bin/activate
```

Comment:
We are somewhat hiding the ansible install from the user. Say a user runs a shell script, that then calls ansible. They will probably get that `ansible-playbook is not a command`. Is this by design? What would be our recommendation to that user. 

The previous install did a `ln -s /usr/local/toolkit/bin/ansible-playbook /usr/bin/ansible-playbook` to make it accessible to others.

Reply:
I like keeping our environment isolated (open to discussion of course).
This should be documented however. We should tell the user how to user and/or copy the environment either in their scripts or post-install. I'll include that in the next push.

Reply:
Among the issues with not having it in the system path is the fact that we would have to rewrite all our playbooks to use a specific path for the ansible binary and the python binary. _This may be a good thing ensuring that our playbooks use a specific release even if other 3rd party playbooks do as they see fit_. But, it's a thing tp be discussed.

This is at least 1 part of what we'd have to look at:

https://docs.ansible.com/ansible/latest/reference_appendices/interpreter_discovery.html

Reply:
Whether we link to `ansible-playbook` from a default PATH location is at least somewhat distinct from the claim that we are or are not isolating the environment. It's not like we'd be copying everything in to a system directory.

Reply:
I added a section to the README that tells the user about this environment and how to use it.

Reply:
@tpdownes, When you mention having to rewrite our playbooks to invoke a specific path, are you thinking of ansible running as runners? I believe that any ansible running as a runner will have the virtualenv activated before running so it will be added to the path (and then deactivated afterward). This will only affect ansible that is being run as a non-runner, such as after startup.

Reply:
I think the main issue is running Ansible by some other means.

(1) I think the README should make more clear that Ansible will only be in the PATH if you activate the virtual environment
(2) If one wants to invoke Ansible via SSH, do we need to set ansible_python_interpreter?

I am happy to address (2) as a new task for after the PR. I do think (1) should be more clear in the PR.

Reply:
(1) has been documented. I also added a sentence on (2) and a suggestion of setting the `ansible_python_interpreter` variable when using ansible in the environment. A task has been added to investigate better ways of handling that.